### PR TITLE
Add iotedge package build for ubuntu 18.04 arm64

### PR DESCRIPTION
**Pull request by ancaantochi**
Added iotedge package build for ubuntu 18.04 arm64 with OpenSSL 1.1

### CODE REVIEW

File: edgelet/build/linux/docker/cross-compile/toolchain.sh [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+       version_path=$DEFAULT_TOOLCHAIN_ARM32_VERSION_PATH
     else
         version=$1
+        version_path=$2
     fi
     local build_tools=gcc-linaro-${version}-x86_64_arm-linux-gnueabihf
 
     echo "Extracting $build_tools"
-    curl -L -o ${build_tools}.tar.xz https://releases.linaro.org/components/toolchain/binaries/latest-7/arm-linux-gnueabihf/${build_tools}.tar.xz
+    curl -L -o ${build_tools}.tar.xz https://releases.linaro.org/components/toolchain/binaries/${version_path}/aarch64-linux-gnu/${build_tools}.tar.xz
```
Comment (philipktlin) [inline edgelet/build/linux/docker/cross-compile/toolchain.sh]: why the download path contains "aarch64"? how about arm?

File: edgelet/build/linux/docker/cross-compile/Dockerfile.ubuntu18.04.aarch64-unknown-linux-gnu [hunk 2]
```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    git \
+    software-properties-common \
+    build-essential \
+    ca-certificates \
+    uuid-dev \
+    curl \
```
Comment (philipktlin) [inline edgelet/build/linux/docker/cross-compile/Dockerfile.ubuntu18.04.aarch64-unknown-linux-gnu]: I remember curl is available in ubuntu:18.04 image.  is it a common practice to put all required software?

Comment (darobs) [reply edgelet/build/linux/docker/cross-compile/Dockerfile.ubuntu18.04.aarch64-unknown-linux-gnu]: Asking for a dependency it already has isn't going to impact anything - I think it's probably better to spell it out.

File: edgelet/build/linux/ubuntu18.04/aarch64/package-iotedged.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+BUILD_REPOSITORY_LOCALPATH=${BUILD_REPOSITORY_LOCALPATH:-$DIR/../../../../..}
+PROJECT_ROOT=${BUILD_REPOSITORY_LOCALPATH}/edgelet
+BUILD_DIR_REL="target/aarch64-unknown-linux-gnu/release"
+BUILD_DIR="$PROJECT_ROOT/$BUILD_DIR_REL"
+
+CARGO_HOME=${CARGO_HOME:-"$HOME/.cargo/"}
+RUSTUP_HOME=${RUSTUP_HOME:-"$HOME/.rustup"}
+IMAGE="edgebuilds.azurecr.io/gcc-linaro-7.4.1-2019.02-x86_64_aarch64-linux-gnu:ubuntu18.04-1"
+
+REVISION=${REVISION:-1}
```
Comment (darobs) [inline edgelet/build/linux/ubuntu18.04/aarch64/package-iotedged.sh]: Are we using this?

**Pull request merged**